tests/app/helpers/test_template_helpers.py

```diff
@@ -103,86 +103,162 @@ def test_footer_context_census_nisra_theme(app: Flask, expected_footer_nisra_the
+                "title": "ONS Surveys",
+            },
+        ),
+        (
+            SurveyType.CENSUS,
+            None,
+            CensusSurveyConfig(),
+            {
+                "title": "Census 2021",
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "titleLogo": "census-logo-en",
+                "titleLogoAlt": "Census 2021",
+            },
+        ),
+        (
+            SurveyType.CENSUS,
+            "Test",
+            CensusSurveyConfig(),
+            {
+                "title": "Test",
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "titleLogo": "census-logo-en",
+                "titleLogoAlt": "Census 2021",
+            },
+        ),
+        (
+            None,
+            None,
+            CensusSurveyConfig(),
+            {
+                "title": "ONS Surveys",
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "titleLogo": "census-logo-en",
+                "titleLogoAlt": "Census 2021",
+            },
+        ),
+        (
+            SurveyType.CENSUS_NISRA,
+            None,
+            CensusNISRASurveyConfig(),
+            {
+                "orgLogo": "nisra-logo",
+                "orgLogoAlt": "Northern Ireland Statistics and Research Agency logo",
+                "titleLogo": "census-logo-en",
+                "titleLogoAlt": "Census 2021",
+                "customHeaderLogo": True,
+                "orgMobileLogo": "nisra-logo-mobile",
+                "title": "Census 2021",
+            },
+        ),
+        (
+            None,
+            None,
+            CensusNISRASurveyConfig(),
+            {
+                "orgLogo": "nisra-logo",
+                "orgLogoAlt": "Northern Ireland Statistics and Research Agency logo",
+                "titleLogo": "census-logo-en",
+                "titleLogoAlt": "Census 2021",
+                "customHeaderLogo": True,
+                "orgMobileLogo": "nisra-logo-mobile",
+                "title": "ONS Surveys",
+            },
+        ),
+        (
+            None,
+            None,
+            SurveyConfig(),
+            {
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "title": "ONS Surveys",
+            },
+        ),
+    ),
+)
+def test_get_page_header_context(
+    app: Flask, theme, survey_title, survey_config, expected
+):
     with app.app_context():
-        survey_config = CensusNISRASurveyConfig()
+        for cookie_name, cookie_value in {
+            "theme": theme,
+            "survey_title": survey_title,
+        }.items():
+            if cookie_value:
+                cookie_session[cookie_name] = cookie_value
+        config = survey_config
```

Comment:
Don't think this is needed, can just use `survey_config`

Reply:
Done, thanks!